configure

```diff
@@ -341,6 +349,23 @@ CXXCMD=`./scripts/find_compiler.sh cxx --ccache=$with_ccache --script-dir=${PWD}
 echo "Setting C compiler to $CCCMD."
 echo "Setting C++ compiler to $CXXCMD."
 
+if [[ $with_clangtidy == 1 ]]; then
+  clangtidy_path=`which clang-tidy || echo ''`
```

Comment:
"``" is actually not preferred. You can install and run `shellcheck` to learn more. `$()` is preferred for readability.